_posts/2023-05-31-mlops-platform-sandbox.md

```diff
@@ -0,0 +1,58 @@
+image:
+  path: /assets/posts/sandbox/sandbox-launch.png
+---
+
+**Last updated:** May 31, 2023
+
+![Screenshots of the MLOps Platform Sandbox](/assets/posts/sandbox/sandbox-launch.png)
+
+We are excited to launch the MLOps Platform Sandbox, a one-click deployment platform for an ephemeral MLOps stack that you can use to run production-ready MLOps pipelines in the cloud. The MLOps Platform Sandbox allows you to create a sandbox with a stack made up of ZenML, Kubeflow, MLflow, and Minio Bucket. The Sandbox comes with pre-built example pipelines that you can run and try out. It provides a seamless experience for you to experiment with these tools without worrying about infrastructure setup and management.
+
+## Simplifying MLOps Platform Deployment
+
+The goal of [ZenML](https://www.zenml.io) is to give ML and MLOps Engineers the ability to pick and choose their preferred infrastructure and tooling to build a platform that fulfills their company's needs. However, deploying ZenML and a rudimentary MLOps platform can be time-consuming for new users. The MLOps Platform Sandbox bridges this gap by providing a one-click deployment platform for a pre-built ephemeral MLOps stack, simplifying the deployment process.
+
+You can sign up with Google and start a demo sandbox. After a few minutes, you'll receive credentials for Kubeflow, Minio, MLflow, and ZenML. You can then use commands like `zenml connect` and `zenml stack set` to set your stacks and `python run.py` to run the pipelines. The sandbox is deleted after 8 hours, and you can choose from a repository of pre-built pipelines to run.
+
+## Why ZenML On Kubeflow / MLflow / Minio?
+
+IMAGE OF STACK GOES HERE
+
+Using ZenML with Kubeflow, MLflow, and Minio is a representative stack as it includes a production-ready orchestrator, an object storage component for data versioning, and a popular experiment tracking tool in machine learning. ZenML's stack recipe system is designed to allow you to easily swap out components and infrastructure based on your company's needs.
+
+## How to Use the Sandbox
+
+The Sandbox provides you with pre-built pipelines that you can easily re-run or modify to suit your needs. If you want to run your own custom code inside your sandbox, there are a few more steps you'll have to do, all of which is [described in our dedicated documentation page](https://docs.zenml.io/user-guide/advanced-guide/sandbox).
+
+To list the available pre-built pipelines with which your sandbox comes, simply use the command `zenml pipeline list`. This will display a list of pipelines, including their build IDs, which you can use to run a specific pipeline.
+
+To run a pipeline with a build, use the command `zenml pipeline run <PIPELINE_NAME> --build_id <BUILD_ID>`. For example, to run the `langchain_pipeline`, you would enter `zenml pipeline run langchain_pipeline --build_id <BUILD_ID>`, replacing `<BUILD_ID>` with the appropriate build ID from the list.
```

Comment:
The langchain pipeline should be replaced with the mlflow example

Reply:
@strickvl Why did you resolve this when it isnt done? :-)

Reply:
done but not pushed...